### PR TITLE
issue_3 util functions that shared by various scripts

### PR DESCRIPTION
**Pull request by zqian**

### CODE REVIEW

File: iTunesUDataPurge/config.properties.template [hunk 1]
```diff
@@ -0,0 +1,3 @@
+iTunesUSiteId=iTunesU_site_id
+sharedSecret=iTunesU_site_sharedSecret
+admin_credential=admin_credential
```
Comment (dovek) [inline iTunesUDataPurge/config.properties.template]: With the new properties file, do the Securities Files on Deluxe need to be updated?

File: iTunesUDataPurge/README.txt [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
-2. RemoveITunesUCourses.java remove the itunesu course from iTunesu server, based on a given list of site ids. Before use, one need to change the program for required credentials and server information. 
+java -cp target/iTunesuDataPurge-1.0-jar-with-dependencies.jar DownloadITunesuFile site_id.txt <download_working_path>
 
- java -classpath . RemoveITunesUCourses site_id_list_file
 
-3. ReadITunesU.java parses the iTunes U showTree file, based on a list of sakai site ids. The result is a tab-spearated text file, showing the total file storage size used by each site page on iTunes U.
+3. RemoveITunesUCourses.java remove the itunesu course from iTunesu server, based on a given list of site ids. Before use, one need to change the program for required credentials and server information. 
 
-java -classpath . ReadITunesU showtree_file site_id_list_file > itunesu_storage_file
+<TODO>
```
Comment (dovek) [inline iTunesUDataPurge/README.txt]: It's not clear to me what this [TODO] is for.

**Comment by bethkirschner** [comment]
This all looks good, but I think there's still more work that needs to be done to be able to directly call a Utils method to generate and display a credential that can be used with Apple's ShowTree API.

Also, this change should reference TLPSO-206 (not a github issue).

File: iTunesUDataPurge/README.txt (hunk 2 above)
Comment (zqian) [inline iTunesUDataPurge/README.txt]: Updated the instruction.

File: iTunesUDataPurge/config.properties.template (hunk 1 above)
Comment (zqian) [inline iTunesUDataPurge/config.properties.template]: Fixed. config.properties.template now has the same parameter settings as those on deluxe server.

**Pull request merged**